### ISSUE TITLE
Failure message not included in XML report.

### ISSUE THREAD

**Issue by ocary** [getgauge/xml-report#16]
Starting with html-report 4.0.7, on failing specs the _failure message_ is not included in the xml-report.

Tested using Gauge 1.0.4 and Gauge 1.0.5.nightly-2019-02-21.  xml-report content was the same for both versions.   Tested using html-report 4.0.7 and 4.0.8.nightly-2019-15

### Expected behavior
Failure message in XML report.

### Actual behavior
test case and failure xml nodes are not included in the xml results file.

### Steps to reproduce
1.  Run a failing spec
2.  Open the xml-report file.
3.  Observe the failure message is not in file.

### Gauge version
```
C:\Users\ucaryne\Downloads>gauge -v
Gauge version: 1.0.5.nightly-2019-02-21
Commit Hash: 612709b

Plugins
-------
csharp (0.10.5)
flash (0.0.1)
html-report (4.0.8.nightly-2019-02-15)
java (0.7.1)
json-report (0.2.2)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.1)
```

Sample file contents; somewhat annonymized.

**Failing spec using html-report 4.0.8.nightly-2019-02-15.  4.0.7 gives the same results**
```
<testsuites>
	<testsuite id="1" tests="1" failures="1" package="C:\filename.spec" time="11.716" timestamp="2019-02-21T14:15:16" name="EndPage" errors="0" hostname="myhost">
		<properties></properties>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>
```

**Failing spec using html-report 4.0.6** 
```
<testsuites>
	<testsuite id="1" tests="1" failures="1" package="C:\filename.spec" time="12.154" timestamp="2019-02-21T13:15:53" name="mytest" errors="0" hostname="myhostname">
		<properties></properties>
		<testcase classname="mytest" name="mytest" time="11.142">
			<failure message="failuremessageisshownhere">failure message</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>
```

**Comment by Debashis9012** [comment]
Able to reproduce this in the below version.


```
<testsuites>
	<testsuite id="1" tests="2" failures="2" package="/Users/example.spec" time="0.001" timestamp="2019-02-26T15:03:21" name="Specification Heading" errors="0" hostname="in-debas.local">
		<properties></properties>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>
```

```
Gauge version: 1.0.4
Commit Hash: 3a9a647

Plugins
-------
html-report (4.0.7)
```

**Comment by NivedhaSenthil** [comment]
Fix available in gauge nightly >= 01.03.2019

**Comment by Debashis9012** [comment]
XML report includes all the failure message. Based on the observation this issue has been 
verified and fixed. Tested with the below version
```
Gauge version: 1.0.5.nightly-2019-03-19
Commit Hash: 7aaf683

Plugins
-------
html-report (4.0.8.nightly-2019-03-08)
java (0.7.2.nightly-2019-03-15)
xml-report (0.2.2.nightly-2019-01-11)
```
